pkgs/development/compilers/ghc/common-hadrian.nix

```
@@ -161,13 +175,7 @@
 assert !enableNativeBignum -> gmp != null;
 
 let
-  src = (if rev != null then fetchgit else fetchurl) ({
-    inherit url sha256;
-  } // lib.optionalAttrs (rev != null) {
-    inherit rev;
-  });
-
-  inherit (stdenv) buildPlatform hostPlatform targetPlatform;
+  inherit (stdenv) hostPlatform targetPlatform;
```

Unnecessary to change.

```suggestion
  inherit (stdenv) buildPlatform hostPlatform targetPlatform;
```

---

Why? `buildPlatform` isn't used for anything in this file.

---

But it might at some point. Having it in scope like the others is less confusing.